tautulli.json

```diff
@@ -0,0 +1,48 @@
+{
+    "Tautulli": {
+        "containers": {
+            "tautulli": {
+                "image": "tautulli/tautulli",
+                "launch_order": 1,
+                "ports": {
+                    "8181": {
+                        "description": "WebUI port. Suggested default: 8181",
+                        "host_default": 8181,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Tautulli configuration. E.g.: create a Share called tautulli-config for this purpose alone.",
+                        "label": "Config Storage"
+                    },
+                    "/plex-config": {
+                        "description": "Choose a Share that holds your Plex log files. This is typically the share that was selected to hold the plex configuration.This will allow Tautulli to read the Plex log files.",
+                        "label": "Plex Config"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid UID to run Tautulli with. It must have full permissions to all Shares mapped in the previous step. This could be the same UID used for your Plex Rock-on",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid GID to use along with the above UID. It(or the above UID) must have full permissions to all Shares mapped in the previous step. This could be the same GID used for your Plex Rock-on",
```

Comment:
Minor typo:
```suggestion
                        "description": "Enter a valid GID to use along with the above UID. It (or the above UID) must have full permissions to all Shares mapped in the previous step. This could be the same GID used for your Plex Rock-on",
```